src/appleseed.qtcommon/widgets/doubleslider.cpp

```diff
@@ -41,15 +41,19 @@ namespace
 DoubleSlider::DoubleSlider(QWidget* parent)
   : QSlider(parent)
 {
-    connect(this, SIGNAL(valueChanged(int)), this, SLOT(setValue(int)));
+    connect(
+        this, &DoubleSlider::valueChanged, 
+        this, static_cast<void (DoubleSlider::*)(const int)>(&DoubleSlider::setValue));
```

Comment:
I'm not sure about this one. Why you cast it ?

Reply:
For this one, when I tried to just use "qOverload" as you recommended, I kept getting an error saying "error: address of overloaded function 'setValue' does not match required type 'qOverload<const int>'". I found this error was because my version of Qt was before 5.6, (see https://stackoverflow.com/questions/16794695/connecting-overloaded-signals-and-slots-in-qt-5). When I built appleseed there was no mention of a minimum Qt version, so I assumed I should make the code work for more Qt versions than less.

Reply:
Should I implement it instead for only Qt 5.6 and above?

Reply:
appleseed supports Qt 5.12 minimum, but I think we forgot to mention it on the wiki for mac devices :/

Reply:
Ah ok, then if appleseed wants to keep supporting a minimum of Qt 5.12, I should leave the static cast. Should I leave a comment in the code that mentions this?

Reply:
Also should I squash the commits?

Reply:
> Ah ok, then if appleseed wants to keep supporting a minimum of Qt 5.12, I should leave the static cast. Should I leave a comment in the code that mentions this?

I thought the static cast was here because qOverload wasn't working on your machine ? It should work with Qt 5.12 so we shouldn't need any static cast.

> Also should I squash the commits?

If you want yes, but we can also do it when merging so it's not required :)